doc/source/whatsnew/v0.17.0.txt

```diff
@@ -429,6 +429,34 @@ Other API Changes
    ``raise ValueError``                All other public methods (names not beginning with underscores)
    ===============================     ===============================================================
 
+- Clarifying ``display.precision`` option to refer to decimal places (:issue:`10451`)
+
+  Earlier versions of pandas would format floating point numbers to have one less decimal place than the value in
+  ``display.precision``.
+
+  .. code-block:: python
+
+    In [1]: pd.set_option('display.precision', 2)
+
+    In [2]: pd.DataFrame({'x': [123.456789]})
+    Out[2]:
+           x
+    0  123.5
+
+  If interpreting precision as "significant figures" this did work for scientific notation but that same interpretation
+  did not work for values with standard formatting. It was also out of step with how numpy handles formatting.
```

Comment:
pls review the doc section http://pandas.pydata.org/pandas-docs/stable/options.html and see if anything needs updating w.r.t. this